torchtext/experimental/datasets/sequence_tagging.py

```diff
@@ -27,16 +27,16 @@ def _setup_datasets(dataset_name,
                     root=".data",
                     vocabs=None,
                     data_select=("train", "valid", "test")):
-    if isinstance(data_select, str):
-        data_select = [data_select]
-    if not set(data_select).issubset(set(("train", "valid", "test"))):
-        raise TypeError("Given data selection {} is not supported!".format(data_select))
-
-    train, val, test = raw.DATASETS[dataset_name](root=root)
+    data_select = check_default_set(data_select, ('train', 'valid', 'test'))
+    if dataset_name == 'UDPOS':
+        train, val, test = raw.DATASETS[dataset_name](root=root)
```

Comment:
This isn't necessary, since you know if you pass "data_select" into raw.DATASETS[dataset_name], you'll get back the datasets in order and only the dataset you actually need

Reply:
Done